### ISSUE TITLE
the problem of train 

### ISSUE THREAD

**Issue by ly-mentaltity** [nv-tlabs/GSCNN#32]
Hello blogger，When I started training the network from scratch, I used four loss functions. The network is very difficult to convergence, four losses have been oscillating, how do you train the network, is the first use of the single loss function at the beginning?

**Comment by ly-mentaltity** [comment]
It is mentioned in the paper that the weight of the first 入1 parameter is 20. Why is the weight before edge_loss in the loss code 20? When the loss is returned, shall we multiply the weight before seg_loss by 20?

**Comment by tovacinni** [comment]
Thanks for your interest in our work!

I'll double check the loss hyperparameters- perhaps I scaled them wrong in the release.

Can you share your training setup with us?
